Update navigation methods and information names to match UI (#651)
mautic/user-documentation: docs/points/points.rst

To add a new action:

> 1. Click **Points > Manage Actions > + New**  - located in the top right corner.

Reviewer: ```suggestion
1. On the left side, navigate to **Points > Manage Actions** and click the **New** button located in the top right corner.
```

Author: Would it be useful to add something like "The New Point Action window will be displayed", so the user immediately knows they've completed the step correctly and this is what the screenshot is referring to?

Reviewer: We can do that. 🙂 
FYI, we don't use passive voice in our docs. Vale can let you know if you need to fix any grammar, including passive voices.

Please run the Vale command in the terminal to make sure there's no style and grammar warnings.
You can find how to work with Vale (and all docs related stuffs) in our [contributing to docs guidelines](https://contribute.mautic.org/en/latest/contributing/contributing_docs.html).

.. image:: images/points-action.png
    :alt: Screenshot of Points action

2. In the main panel, there are five boxes for key information. Enter the appropriate information.

   * **Name** - The name of your action. This is how the action displays in your list of actions, so choose an identifiable name.

   * **Description** - Add a description to help you find certain actions. There may be more actions which are similar or more in-depth.

   * **Change Points (+/-)** - The value change to set for the action. The ``+`` isn't necessary when adding Points. When subtracting Points, add the ``-`` symbol.

>    * **Actions taken by contact** - This is the behavior or action the Contact must complete to trigger the action.

Reviewer: I understand that on the form, the "Contact" is in lowercase. However, Contact is Mautic's feature and that's why Vale gives warning. So, let's change this to uppercase.

I think we can also adjust the wording.

```suggestion
   * **Actions taken by Contact** - The specific Contact activity required to trigger the point action.
```

>    * **Point Group** - This is the Point Group you want the action applied to.

Reviewer: Let's reword this for clarity.

```suggestion
   * **Point group** - Identify the target group for the action. An empty selection applies the action to all main Contact points by default.
```

Author: Is "Point group" not a feature too? should be uppercase, no?

Reviewer: "Points" is our feature. But "group" is not.

Author: Is it not the same as this? 
<img width="400" height="144" alt="mautic-point-groups" src="https://github.com/user-attachments/assets/28d9dbd4-4a1f-469f-936c-27e7bae15552" />

Reviewer: It's the same. We might haven't updated that because we use [sentence instead of title case](https://www.scribbr.com/academic-writing/sentence-case/) for our docs.

The quick way to test if a word is probably a Mautic feature is by running Vale. If it thinks that a word is Mautic's feature, it gives you a warning with explanation.

Author: Ok, thanks for the clarification. Part of my confusion was because I was getting a suggestion in Vale:

<img width="613" height="73" alt="vale-suggestion-point-group" src="https://github.com/user-attachments/assets/e396712a-f859-45a0-844c-33602dae067d" />

Reviewer: Hey @mheavey2,

You're right. It should be "**Point Groups**". I've confirmed this to our team.

What happened was, I checked the Vale on our developer docs, and it didn't give any suggestion for "group" in lowercase. We now found out thatw e need to update the Vale dictionary in our dev docs. 🙂 

Sorry for the confusion!

Author: No worries! All part of the learning :)

3. On the right side is more information:

   * **Category** - Organize your Point Actions based on their goals, Campaigns, etc. For more information, see :doc:`Categories</categories/categories-overview>`. All Points accumulate on a Contact record, regardless of Category. There is one Points score for each Contact.
   

>    * **Projects** - Organize your Point Actions based on their Campaigns project.

Reviewer: Where do you see "Projects"? Because I don't see it anywhere on the right side.

<img width="311" height="572" alt="Image" src="https://github.com/user-attachments/assets/56f00b03-d714-41a8-89a9-50e3699f7740" />

Author: Sorry, my bad, I looked at the wrong version.

To add a new action:

> 1. On the left side, navigate to **Points > Manage Actions** and click the **New** button located in the top right corner to open the New Point Action window.

Reviewer: ```suggestion
1. On the left side, navigate to **Points > Manage Actions** and click the **New** button located in the top right corner to open the **New Point Action** window.
```

.. image:: images/points-action.png
    :alt: Screenshot of Points action

> 2. In the main panel, there are five boxes for key information. Enter the appropriate information.

Reviewer: Let's omit the number to give flexibility for Mautic versions which have different number of fields:

```suggestion
2. In the main panel, fill in the following fields:
```

   * **Name** - The name of your action. This is how the action displays in your list of actions, so choose an identifiable name.

   * **Description** - Add a description to help you find certain actions. There may be more actions which are similar or more in-depth.

   * **Change Points (+/-)** - The value change to set for the action. The ``+`` isn't necessary when adding Points. When subtracting Points, add the ``-`` symbol.

>    * **Actions taken by Contact** - The specific Contact activity required to trigger the point action.

Reviewer: It's "Action", not "Actions".

```suggestion
   * **Action taken by Contact** - The specific Contact activity required to trigger the Point Action.
```

Once a Contact has accumulated a Point total, you may want to trigger an action with the Contact. You may create multiple triggers for different Point values.

To add a new trigger:

> 1. On the left side, navigate to **Points > Manage Triggers** and click the **New** button located in the top right corner to create a new point trigger.

Reviewer: ```suggestion
1. On the left side, navigate to **Points > Manage Triggers** and click the **New** button located in the top right corner to create a new Point Trigger.
```

.. image:: images/points-trigger.png
    :alt: Screenshot of Points trigger

Creating Point Triggers is like creating Point Actions. The **Name**, **Description**, **Point Group**, **Category** and **Active** options are all the same. 

> 2. The main panel also includes additional options. Enter the appropriate information.

Reviewer: ```suggestion
2. The main panel includes the following inputs to enter the appropriate information:
```

> 2. The main panel also includes additional options. Enter the appropriate information.

Reviewer: We can combine this into one for better flow and clarity:

```suggestion
2. Complete the main panel information. Use the **Name**, **Description**, **Point Group**, and **Category** fields as you do for Point Actions. The **Active** toggle and **Active and Activate/Deactivate at date/time** settings also work the same way. Then, fill in these extra fields:
```

    * **Minimum number of Points** - The minimum number of Points required to trigger the Point Action.

>     * **Contact color** - Designate a specific color to Contacts that have at least the minimum number of Points specified. 

Reviewer: We can simplify this to facilitate people who are non-native English:

```suggestion
    * **Contact color** - This sets a highlight color for Contacts who earn at least the minimum number of Points.
```

>     * **Trigger for existing applicable Contacts upon saving** - To apply the trigger to existing Contacts that have at least the minimum number of Points specified select **Yes**.

Reviewer: ```suggestion
    * **Trigger for existing applicable Contacts upon saving** - Select **Yes** to apply this trigger to Contacts who already have the minimum number of Points.
```

Once a Contact has accumulated a Point total, you may want to trigger an action with the Contact. You may create multiple triggers for different Point values.

To add a new trigger:

> 1. Click **Points > Manage Triggers > + New**  - located in the top right corner. This opens in the **Details** tab.

Reviewer: Because there's only a single point, I think we better squash this into one sentence.

```suggestion
On the left side, navigate to **Points > Manage Triggers** and click the **New** button located in the top right corner to create a new point trigger. 
```

.. image:: images/points-trigger.png
    :alt: Screenshot of Points trigger

> Creating Point Triggers is like creating Point Actions. The **Name**, **Description**, **Point Group**, **Category**,  **Projects** and **Active** options are all the same. The trigger fires based on the minimum number of Points. Set a number and decide if you want to **Trigger for existing applicable Contacts upon saving - if activated**. 

Reviewer: There are a couple of things here:

- I still don't see "Projects" anywhere in New Trigger
- Unlike point actions, there's "Contact color" in point triggers. We need to include and explain that.
